MAINTAINERS.yml

```diff
@@ -5125,6 +5127,7 @@ West:
   maintainers:
     - nzmichaelh
     - kholia
+    - VynDragon
```

Comment:
3 maintainers is probably too much. Always better to only have a couple individuals actually on point

Reply:
Done